SuperBuild/External_LibArchive.cmake

```diff
@@ -38,7 +38,7 @@ if((NOT DEFINED LibArchive_INCLUDE_DIR
 
   ExternalProject_SetIfNotDefined(
     ${CMAKE_PROJECT_NAME}_${proj}_GIT_REPOSITORY
-    "${EP_GIT_PROTOCOL}://github.com/Slicer/LibArchive.git"
+    "${EP_GIT_PROTOCOL}://github.com/phcerdan/libarchive"
```

Comment:
change phcerdan for slicer once the branch is created from https://github.com/Slicer/libarchive/pull/1